### PR TITLE
graphql indexing not needed

### PR DESCRIPTION
**Pull request by omurbekjk**

### CODE REVIEW

File: services/config.rb [hunk 1]
```diff
@@ -32,9 +32,14 @@
   operators [""]
   raise_when [""]
   id_map "static.no_op"
-  meta_rule_query "{ query(func: has(key)) @filter(%<key_filter>s AND NOT has(relates_to))  { objectId }}"
+  meta_rule_query <<~QUERY
+  { 
+    query(func: %<key_filter>s) @filter(NOT has(relates_to))  
+    { objectId }
```
Comment (armaneous) [inline services/config.rb]: Opening curly brace should be at end of previous line and please add `%<default_predicates>s`

File: services/config.rb [hunk 2]
```diff
@@ -32,9 +32,14 @@
...
+  }
+  QUERY
   meta_rule_node_triggers ({
-      'key' => []
+      'key' => ['']
```
Comment (armaneous) [inline services/config.rb]: Why is there an empty string in the property trigger?

File: services/config.rb [hunk 3]
```diff
@@ -57,9 +62,28 @@
   operators ["", "=="]
   raise_when ["", false]
   id_map "modifiers.key_id"
-  meta_rule_query "{ query(func: has(key)) @filter(%<key_filter>s) @cascade { objectName  relates_to @filter(has(key_rotation_status) AND eq(key_rotation_enabled, false)){ location } } }"
+  meta_rule_query <<~QUERY
+  { 
+    keys as var(func: %<key_filter>s) @cascade 
+    {
```
Comment (armaneous) [inline services/config.rb]: Same about curly brace...

File: services/config.rb [hunk 4]
```diff
@@ -57,9 +62,28 @@
   operators ["", "=="]
   raise_when ["", false]
   id_map "modifiers.key_id"
-  meta_rule_query "{ query(func: has(key)) @filter(%<key_filter>s) @cascade { objectName  relates_to @filter(has(key_rotation_status) AND eq(key_rotation_enabled, false)){ location } } }"
+  meta_rule_query <<~QUERY
+  { 
+    keys as var(func: %<key_filter>s) @cascade 
+    {         
+      keys_rotation as relates_to @filter(has(key_rotation_status))
+      {
```
Comment (armaneous) [inline services/config.rb]: Same about curly brace...

File: services/config.rb [hunk 5]
```diff
@@ -57,9 +62,28 @@
...
+        krs as key_rotation_status
+      }
+    }
+    
+    query(func: uid(keys)) @cascade 
+    {
+      objectName
```
Comment (armaneous) [inline services/config.rb]: Same about curly brace and please add `%<default_predicates>s`

File: services/config.rb [hunk 6]
```diff
@@ -57,9 +62,28 @@
...
+      relates_to @filter(uid(keys_rotation) AND eq(val(krs), false) ) 
+      {
+        objectName
```
Comment (armaneous) [inline services/config.rb]: Same about curly brace and please add `%<default_predicates>s`

File: services/config.rb [hunk 7]
```diff
@@ -57,9 +62,28 @@
...
+      }
+    }    
+  }
+  QUERY
   meta_rule_node_triggers ({
-      'key' => [],
+      'key' => [''],
```
Comment (armaneous) [inline services/config.rb]: Same about empty string...

File: services/config.rb (hunk 2 above)
Comment (omurbekjk) [reply services/config.rb]: it will trigger when only kms node changes, or it should be empty array?

**Pull request merged**